### PR TITLE
Add configs for MiroThinker v0.1 SFT

### PR DESCRIPTION
**Pull request by jenny-miromind**
In this PR, I added the configs for reproducing the MiroThinker v0.1 SFT model training.

These configs cover three different sizes of MiroThinker models. The training data used is the `MiroVerse-v0.1-all` subset from [MiroVerse-v0.1](https://huggingface.co/datasets/miromind-ai/MiroVerse-v0.1), with the split set to `train`.

| Hyperparameter | MiroThinker-8B-SFT-v0.1  | MiroThinker-14B-SFT-v0.1 | MiroThinker-32B-SFT-v0.1 |
| -------------- | -------- | --------- | --------- |
| Epochs         | 4        | 3         | 3         |
| Learning Rate  | 4e-5     | 4e-5      | 4e-5      |
| Weight Decay   | 0.1      | 0.1       | 0.1       |
| Packed Data    | Enabled  | Enabled   | Enabled   |
| Context Length | 40k      | 40k       | 40k       |
| Batch Size     | 128      | 128       | 128       |
| Clip Grad Norm | 1.0      | 1.0       | 1.0       |
| Warmup Ratio   | 0.1      | 0.1       | 0.1       |

### CODE REVIEW

File: recipes/configs/mirothinker_v0_1/sft/mirothinker_v0_1_14B_qwen3_40k_bs128_pack.yaml [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+# Memory management
+enable_activation_checkpointing: True  # True reduces memory
+enable_activation_offloading: False  # True reduces memory
+
+# Tokenizer
+tokenizer:
+  _component_: mirotrain.models.qwen3.qwen3_tokenizer_auto
+  path: ./models/Qwen3-14B
+  max_seq_len: 40960
```
Comment (jenny-miromind) [inline recipes/configs/mirothinker_v0_1/sft/mirothinker_v0_1_14B_qwen3_40k_bs128_pack.yaml:16]: The context length we used for training is 40k.

File: recipes/configs/mirothinker_v0_1/sft/mirothinker_v0_1_14B_qwen3_40k_bs128_pack.yaml [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+    model-00008-of-00008.safetensors,
+  ]
+  recipe_checkpoint: null
+  output_dir: ${output_dir}
+  model_type: QWEN3
+resume_from_checkpoint: False
+
+## Fine-tuning arguments
+batch_size: 1
+epochs: 3
```
Comment (jenny-miromind) [inline recipes/configs/mirothinker_v0_1/sft/mirothinker_v0_1_14B_qwen3_40k_bs128_pack.yaml:57]: The 8B model is trained for 4 epochs, while the other sizes (14B / 32B) are trained for 3 epochs.

File: recipes/configs/mirothinker_v0_1/sft/mirothinker_v0_1_14B_qwen3_40k_bs128_pack.yaml [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+gradient_accumulation_steps: 1  # Use to increase effective batch size
+
+loss:
+  _component_: mirotrain.modules.loss.LigerLinearCrossEntropyLoss
+
+optimizer:
+  _component_: torch.optim.AdamW
+  fused: True
+  weight_decay: 0.1
+  lr: 4e-5
```
Comment (jenny-miromind) [inline recipes/configs/mirothinker_v0_1/sft/mirothinker_v0_1_14B_qwen3_40k_bs128_pack.yaml:67]: All models use a learning rate of 4e-5.

**Review by pax-sun — APPROVED** [review]
